pkg/virt-handler/vm_test.go

```diff
@@ -1151,6 +1151,35 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(blockMigrate).To(BeTrue())
 			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared HostDisk")))
 		})
+
+		It("should allow to migrate with adding default networks", func() {
```

Comment:
I don't understand the sentence. Maybe "should allow to migrate with the default network configuration"?
 
Anyway, this should fail. The default binding is bridge and we don't allow to migrate it with that. So there is probably a missing validation step in the code.

Reply:
Changed this with an additional logic to check if networks and interfaces are not empty.